tests/test_models.py

```diff
@@ -791,3 +791,49 @@ def test_fileinstance_validation(app, db, dummy_location):
     f = FileInstance.create()
     f.set_uri('x' * 255, 1000, 1000)  # Should not raise
     pytest.raises(ValueError, f.set_uri, 'x' * 256, 1000, 1000)
+
+
+def test_object_version_tags(app, db, dummy_location):
+    """Test object version tags."""
+    f = FileInstance(uri="f1", size=1, checksum="mychecksum")
+    db.session.add(f)
+    db.session.commit()
+    b = Bucket.create()
+    obj1 = ObjectVersion.create(b, "test").set_file(f)
+    ObjectVersionTag.create(obj1, "mykey", "testvalue")
+    ObjectVersionTag.create(obj1, "another_key", "another value")
+    db.session.commit()
+
+    # Duplicate key
+    pytest.raises(
+        Exception, ObjectVersionTag.create, obj1, "mykey", "newvalue")
+
+    # Test get
+    assert ObjectVersionTag.query.count() == 2
+    assert ObjectVersionTag.get(obj1, "mykey").value == "testvalue"
+    assert ObjectVersionTag.get_value(obj1.version_id, "another_key") \
+        == "another value"
+    assert ObjectVersionTag.get_value(obj1, "invalid") is None
+
+    # Test delete
+    ObjectVersionTag.delete(obj1, "mykey")
```

Comment:
Can you also add a test for the cascade of ``ObjectVersion.delete()``. I.e. that if you delete an ``ObjectVersion``, the tags are also deleted.